hooks/599-cleanup-packages.chroot

```diff
@@ -4,4 +4,8 @@ set -e
 echo Removing pipewire
 # remove packages that were installed due to dependencies but that we
 # must remove.
-dpkg -r --force-all pipewire pipewire-bin wireplumber pipewire-pulse libspa-0.2-modules libpipewire-0.3-modules
+dpkg -r --force-all pipewire wireplumber pipewire-pulse
+
```

Comment:
Could you add a comment explaining what that regex do, please? Maybe we know now what it does, but our future "we" probably won't 😇

Reply:
Now done